hydromt/models/model.py

```diff
@@ -1466,39 +1464,6 @@ def crs(self) -> CRS:
         """Returns coordinate reference system embedded in region."""
         return self.region.crs
 
-    # test methods
-    def _test_equal(self, other, skip_component=None) -> Tuple[bool, Dict]:
-        """Test if two models including their data components are equal.
```

Comment:
This method is very useful so I would like to see it transfered. Maybe here just for each component create their own ``test_equal`` method and here like ``Model.write`` just loop through the components.